docs/user/sending-notifications.rst

```diff
@@ -78,7 +77,7 @@ Additional ``notify`` Keyword Arguments
 ================= ======================================================
 
 Passing Extra Data to Notifications
------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
What do you think about adding a call to `notify()` in the following example to make it clearer?